cms/djangoapps/contentstore/management/commands/set_advanced_fields_all_courses.py

```diff
@@ -0,0 +1,248 @@
+"""
+Script for updating an Advanced Setting across all courses.
+
+This script is setup to only update the `modulestore` courses.
+"""
+
+import copy
+import json
+import logging
+
+from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
+from django.core.management import CommandError
+from django.core.management.base import BaseCommand
+
+from .utils import user_from_str
+from ....contentstore.views.course import update_course_advanced_settings
+
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.django import modulestore
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Duraration:
+        When running this against all courses it may take considerable more time because
+        the course publishes out. This is due to the `update_course_advanced_settings` call.
+
+    Invoke with:
+
+        # Single Course
+        # Set the following advanced settings:
+        # `invitation_only` to `true`
+        # `catalog_visilibilty` to `about`
+        python manage.py cms --settings=devstack_docker set_advanced_fields_all_courses \
```

Comment:
I believe `--settings=devstack_docker` is no longer required to be specified, and is definitely wrong on production environments anyways, so should we omit that from this docstring?